login-workflow/example/src/screens/demo-components/OrganizationDetailsScreenBaseDemo.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { OrganizationDetailsScreenBase } from '@brightlayer-ui/react-auth-workflow';
+import { stateList } from './country-state';
+
+export type StateType = {
+    name: string;
+    isoCode: string;
+    countryCode: string;
+};
+
+const countryList = [
+    {
+        name: '',
+        isoCode: '',
+    },
+    {
+        name: 'Afghanistan',
+        isoCode: 'AF',
+        flag: '🇦🇫',
+        phonecode: '93',
+        currency: 'AFN',
+    },
+    {
+        name: 'Aland Islands',
+        isoCode: 'AX',
+        flag: '🇦🇽',
```

Comment:
Didn't see the use of this other isoCode, flag, phonecode and currency. Are we going to use this on another screen?

Reply:
These constants were provided from the DERMS project. I simply copied them so there may be issues with the data set or unused keys